src/main/fc/fc_msp.c

```diff
@@ -1338,7 +1340,24 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
 #endif
         break;
-
+#ifdef USE_RCSPLIT
+    case MSP_RCSPLIT_BOXNAMES:
+        serializeRCSplitBoxNamesReply(dst);
+        break;
+    case MSP_RCSPLIT_BOXIDS:
+        serializeRCSplitBoxIdsReply(dst);
+        break;
+    case MSP_RCSPLIT_MODE_RANGES:
+        for (int i = 0; i < MAX_RC_SPLIT_MODE_ACTIVATION_CONDITION_COUNT; i++) {
+            const modeActivationCondition_t *mac = rcSplitModeActivationConditions(i);
+            const box_t *box = findRCSplitBoxByBoxId(mac->modeId);
+            sbufWriteU8(dst, box->permanentId);
+            sbufWriteU8(dst, mac->auxChannelIndex);
+            sbufWriteU8(dst, mac->range.startStep);
+            sbufWriteU8(dst, mac->range.endStep);
+        }
+        break;
+#endif
```

Comment:
![adeeec2c-6afc-401d-86a8-18ef362d4f9f](https://user-images.githubusercontent.com/1505751/26972054-8307dae0-4d42-11e7-8d11-edb7b2b1f383.png)
![bf1a0bfb-f12e-4ed7-8454-43201ed190bd](https://user-images.githubusercontent.com/1505751/26972055-835652e2-4d42-11e7-8b56-b2db52a5682b.png)

hihi, please take a look of above images, this is why I need add new msp commands